src/pages/program/[slug]/edit-profile.tsx

```diff
@@ -0,0 +1,179 @@
+
+  const { profileId, tagsJson } = currentProfile;
+  const { firstName, lastName, profilePictureUrl } = data.getMyUser;
+  const tagsList = getTagsFromJson(tagsJson);
+  const tags = tagsList?.slice(0, 3).map((tag: string, index: number) => (
+    <div className="rounded-md bg-tertiary m-1 p-1" key={index}>
+      {tag}
+    </div>
+  ));
+  const moreTag = (
+    <div className="rounded-md border-primary border m-1 p-1">
+      + {tagsList.length - 3} more
+    </div>
+  );
+
+  switch (authorizationLevel) {
+    case AuthorizationLevel.Mentor:
+    case AuthorizationLevel.Mentee:
+      const isMentor = authorizationLevel === AuthorizationLevel.Mentor;
+      return (
+        <div className="h-screen bg-tertiary flex flex-col items-center py-20">
+          <div className="w-200 flex flex-col items-center ">
+            <div className="flex justify-between items-center w-full">
+              <Text h2 b>
+                Edit My {isMentor ? "Mentor" : "Mentee"} Profile
+              </Text>
+              <div className="w-12"></div>
+              <div className="flex">
+                <Button
+                  size="small"
+                  disabled={!modified}
+                  onClick={() => {
+                    const updateProfileInput: UpdateProfileInput = {
+                      profileJson: JSON.stringify(profileJson),
+                      bio: bio,
+                    };
+                    updateProfile({
+                      variables: {
+                        profileId: profileId,
+                        data: updateProfileInput,
+                      },
+                    }).then(() => {
+                      if (refetchCurrentProfile) refetchCurrentProfile();
+                      setModified(false);
+                    });
+                  }}
+                >
+                  Save
+                </Button>
+              </div>
+            </div>
+            <div className="h-8"></div>
+            <Card className="flex flex-col w-80 p-6 items-center border-0">
+              <div className="h-40 w-40 rounded-full bg-tertiary">
+                <img src={profilePictureUrl}></img>
+              </div>
+              <div className="h-4"></div>
+
+              <Text b className="text-body-1 text-center">
+                {firstName} {lastName}
+              </Text>
+              <div className="h-4"></div>
+
+              <div className="flex flex-wrap justify-center">
+                {tags}
+                {tags?.length > 3 ? moreTag : <div></div>}
+              </div>
+              <div className="h-4"></div>
+
+              <textarea
+                value={bio}
+                onChange={(e) => {
+                  setModified(true);
+                  setBio(e.target.value);
+                }}
+                className="p-2 shadow-sm focus:ring-secondary focus:border-primary mt-1 block sm:text-sm border border-secondary rounded-md"
```

Comment:
make resize-none and probably just have it scroll + make sure the card sizes match so that preview is accurate (?)